apache-maven/src/assembly/maven/conf/maven.properties

```diff
@@ -2,20 +2,40 @@
 # Maven user properties
 #
 
-maven.user.home = ${user.home}/.m2
+maven.install.conf = ${maven.home}/conf
+maven.user.conf    = ${user.home}/.m2
+maven.project.conf = ${session.rootDirectory}/.mvn
 
-# Required files to load
-${includes} =
+# Space separated list of files to load.
+# If the first character is a question mark, the load will silently fail if the file does not exist.
+${includes} = ?${maven.user.conf}/maven.properties, \
+              ?${maven.project.conf}/maven.properties
 
-# Optionally load user properties and project user properties
-${optionals} = ${maven.user.home}/maven.properties \
-               ${session.rootDirectory}/.mvn/maven.properties
 
-# Default files
+#
+# Settings
+#
+# Define the default three levels for settings.
+# The '-is' flag will override the 'maven.install.settings' property.
+# The '-ps' flag will override the 'maven.project.settings' property.
+# The '-s' flag will override the 'maven.user.settings' property.
+maven.install.settings = ${maven.install.conf}/settings.xml
```

Comment:
Attention: This must not be confused with the Maven Install Plugin, hence: `maven.installation.<prop>`